Intro/Intro.md

```diff
@@ -11,7 +11,7 @@ structured information in the form of self-contained data (as opposed to a Word
 usually in JSON or XML format. RavenDB is, essentially, a database for storing and working with JSON data.
 
 RavenDB can run on a single node (suitable for development or for small applications) or on a cluster of nodes
-(high availability, load balancing, geo distribution of data and work, etc.). A single cluster can host multiple
+(high availability, load balancing, geo distribution of data and work, etc.). A single node can host multiple
```

Comment:
By we are talking immediately afterward about multiple nodes, that would be confusing.